config/locales/it.yml

```diff
@@ -9,16 +9,28 @@ it:
   buttons:
     login: "Login"
   payment_type:
-    not_yet: 'Non ancora'
-    paypal: 'Si. Con PayPal'
-    bank_transfer: 'Si. Con Bonifico'
+    not_yet: "Non ancora"
+    paypal: "Si. Con PayPal"
+    bank_transfer: "Si. Con Bonifico"
   activerecord:
     errors:
       models:
         user:
           attributes:
             email:
-              taken: 'Email già registrata'
+              taken: "Email già registrata"
   documents: "Documentazione"
   download_payment_recipe: "Download Payment recipe"
   download_document: "Download document"
+  admin_mailer:
```

Comment:
per ora va bene in un unico file. devise ha il suo file ma il resto possiamo metterlo su un unico file